app/models/remote_theme.rb

```diff
@@ -16,6 +17,8 @@ class ImportError < StandardError; end
     joins("JOIN themes ON themes.remote_theme_id = remote_themes.id").where.not(remote_url: "")
   }
 
+  validates_format_of :minimum_discourse_version, :maximum_discourse_version, with: /\A\d+\.\d+\.\d+(\.beta\d+)?\z/, allow_nil: true
```

Comment:
I feel like this Regexp might find a better home in `lib/version.rb`. Would encourage more re-use.